inginious/backend/topic_priority_queue.py

```diff
@@ -36,10 +36,11 @@ def get(self, topics=None):
         :return: the smallest elements that fits in one of the topics
         :raises: queue.Empty exception if the queue has no elements that fits in any of the topics
         """
+        agent_ssh = topics.ssh_allowed
```

Comment:
TopicPriorityQueue is a reusable component, that could even be moved to the `common` package; it shouldn't be aware of the content of the objects given to it.

It's a bad smell, these exception should be handled elsewhere.